3rdparty/hal_rvv/hal_rvv_1p0/norm.hpp

```diff
@@ -1008,6 +1008,7 @@ inline int norm(const uchar* src, size_t src_step, const uchar* mask, size_t mas
         sizeof(int),     sizeof(float),
         sizeof(int64_t), 0,
     };
+    assert(elem_size_tab[depth]);
```

Comment:
Looks like the assert will be disabled in regular release builds: https://en.cppreference.com/w/cpp/error/assert.
Why not just CV_Assert? It's defined in `opencv2/core/base.hpp`

Reply:
Ok